components/CircleContainer/Circle.tsx

```diff
@@ -85,6 +86,9 @@ const Circle: React.FC<CircleProps> = ({
             <source src={openProject.mp4} type="video/mp4" />
           </StyledVideo>
         )}
+        {/* To Tyler: I thought that this would force these videos to preload, but I'm still noticing that the
+        video isn't appearing immediately when a project is clicked. Note that hidden={true} simply adds a huge negative
+        margin that puts the video off screen */}
```

Comment:
you'll have to add a listener onto this video element, that listens for when the video is ready to be played, and inbetween the click to open the project, and the time the video loads, you need to display some kind of loading state. Preload only works with a static source.